policyengine_us/parameters/gov/states/al/tax/income/adjustments.yaml

```diff
@@ -0,0 +1,20 @@
+description: Alabama provides these adjustments for the state adjusted gross income.
+values:
+  2021-01-01:
+  # IRA distributions prior to 1982 are not deductable
+    - roth_ira_distributions # Line 1 
+    - regular_ira_distributions # Line 1 
```

Comment:
@MaxGhenis do we represent the Keogh RP?

source - https://www.revenue.alabama.gov/ultraviewer/viewer/basic_viewer/index.html?form=2023/01/22f40bk.pdf

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/7f518777-0e4e-4ad3-8a1e-7919bb6f4d37)